src/Kiosk.Api/Controllers/NewsController.cs

```diff
@@ -57,13 +57,16 @@ public NewsController(ILogger logger, INewsService newsService)
         CancellationToken cancellationToken,
         [FromQuery, Required]
         Language language,
-        [FromQuery] Source? source=null)
+        [FromQuery] Source? source=null,
+        [FromQuery(Name = "page")] int page = 1,
+        [FromQuery(Name = "itemsPerPage")] int itemsPerPage = 30
+        )
     {
         try
         {
-            var newsList = await _newsService
-                .GetTranslatedListOfNews(source, language, cancellationToken);
-            return newsList is null ? NoContent() : Ok(newsList);
+            var (content, pagination) = await _newsService
+                .GetTranslatedListOfNews(source, language, page, itemsPerPage, cancellationToken);
+            return content is null ? NoContent() : Ok(new { content, pagination});
```

Comment:
it would be way better to make Pagination a generic interface with `content` property. That way we could represent a response dto in a better way